File: app/src/main/java/tw/y_studio/ptt/Utils/DebugUtils.java
package tw.y_studio.ptt.Utils;

import android.util.Log;

import tw.y_studio.ptt.BuildConfig;

public class DebugUtils {
    public static final boolean Debug = BuildConfig.DEBUG;
    public static final boolean useApi = true;
    public static void Log(String title,String message){
        if(Debug){
            Log.d(title,message);
        }
    }
}

File: app/src/main/java/tw/y_studio/ptt/utils/DebugUtils.kt
@file:JvmName("DebugUtils")

package tw.y_studio.ptt.utils

import tw.y_studio.ptt.BuildConfig

const val useApi = true

fun Log(title: String, message: String) {
    if (BuildConfig.DEBUG) {
        android.util.Log.d(title, message)
    }
}
